modules/install-booting-from-an-iso-over-http-redfish.adoc

```diff
@@ -53,3 +53,17 @@ $ curl -k -u <bmc_username>:<bmc_password> -d '{"ResetType": "ForceRestart"}' -H
 ----
 $ curl -k -u <bmc_username>:<bmc_password> -d '{"ResetType": "On"}' -H 'Content-type: application/json' -X POST <host_bmc_address>/redfish/v1/Systems/System.Embedded.1/Actions/ComputerSystem.Reset
 ----
+
+. Eject the ISO by running the following command:
++
+[source,terminal]
+----
+$ curl -k -u <bmc_username>:<bmc_password> -d '{}' -H "Content-Type: application/json" -X POST <host_bmc_address>/redfish/v1/Managers/iDRAC.Embedded.1/VirtualMedia/CD/Actions/VirtualMedia.EjectMedia
+----
+
+. Reset the boot mode by running the following command:
++
+[source,terminal]
+----
+$ curl -k -u <bmc_username>:<bmc_password> -X PATCH -H 'Content-Type: application/json' -d '{"Boot": {"BootSourceOverrideTarget": "None", "BootSourceOverrideMode": "Legacy", "BootSourceOverrideEnabled": "Once"}}' <host_bmc_address>/redfish/v1/Systems/System.Embedded.1
```

Comment:
Please check if the `"Legacy"` value is correct in `"BootSourceOverrideMode": "Legacy"`